### PR TITLE
7 social media logos done

### PR DESCRIPTION
**Pull request by evnxprk**
# 7 social media logos done

## Description
I have added seven social media logos and added the social media website in social-links.json

---
## Issue Ticket Number
Fixes #(issue_number)

---
## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

---
# Checklist:
- [x] I have followed the contributing guidelines of this project as mentioned in [CONTRIBUTING](./docs/CONTRIBUTING)
- [x] I have checked to ensure there aren't other open [Pull Requests](https://github.com/Param302/SocialRepo/pulls) for the same update/change?
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes needed to the documentation

### CODE REVIEW

File: src/assets/logos/discord.png
Comment (Param302) [inline src/assets/logos/discord.png:1]: This discord icon seems to be of old version, please add the latest icon.

File: src/assets/logos/github.png
Comment (Param302) [inline src/assets/logos/github.png:1]: The github icon is black, not blue, please add the correct one 😅

File: src/social-links.json [hunk 1]
```diff
@@ -1,4 +1,35 @@
...
+    "social_media_name": "spotify",
+    "spotify": "https://wwww.spotify.com/"
+  },
+  {
+    "social_media_name": "github",
+    "github": "https://www.github.com/"
+  },
+  {
+    "social_media_name":"gitlab",
+    "gitlab": "https://www.gitlab.com/"
```
Comment (Param302) [inline src/social-links.json:32]: I guess, there's a confusion in the syntax 😅
All the links should be in one dictionary. Check syntax below 👇
```json
{
    "social_media_name": "link",
	"github": "https://github.com/"				// Example
    "linkedin": "https://www.linkedin.com/in/"	// Example
	
}
```

Like this.
The first line is just for the example, representing the **key** should contain the `social_media_name` and the **value** should contain `user profile prefix URL`.

Please add the links as specified in the above manner. Also, the keys should be in sorted alphabetically, so we can maintain an order.

I hope you are clear with the changes, if you have any confusions, feel free to ask.

_Happy Contributing_ 😊 🙌

**Comment by Param302** [comment]
> # Title
> ## Description
> ## Issue Ticket Number
> 
> Fixes #(issue_number)

Also, specify the Title of the PR, Description of what have you changed, and Issue number so we can keep track off.

**Review by Param302 — CHANGES_REQUESTED** [review]
I have added the changes in comments.
Please check them and change accordingly.

And, I hope you are clear with the changes, if you have any confusions, feel free to ask.

_Happy Contributing_ 😊 🙌

**Comment by evnxprk** [comment]
I have finished the changes that you've requested. As for the issue number, I'm unsure where that is located so I left it blank.

File: src/assets/logos/github.png
Comment (Param302) [inline src/assets/logos/github.png:1]: The github logo is a little odd.
You can use github logo from [here](https://github.com/logos).

**Review by Param302 — CHANGES_REQUESTED** [review]
Hey,
Just change the github logo as specifiec. Rest is all good, then I will merge the PR.

_Thanks for putting so much effort and contributing_

**Comment by evnxprk** [comment]
I've fixed the github logo, let me know if this change is good to go.

**Comment by Param302** [comment]
Hey @evnxprk ,

## Amazing work done! ⭐
I am merging your PR to main branch.
Thanks for your contribution in **SocialRepo**

_Keep Contributing_ 🙌

**Pull request merged**